Dockerfile

```diff
@@ -8,11 +8,12 @@ RUN set -x \
     && addgroup -g 101 -S indexer \
     && adduser -S -D -u 101 -h /home/indexer -s /sbin/nologin -G indexer -g indexer indexer
 
-RUN apk add --no-cache --update git ca-certificates
+RUN apk add --no-cache --update git ca-certificates yaml-dev build-base
 
 ADD requirements.txt /
-RUN pip install --upgrade pip
-RUN pip install -r /requirements.txt
+RUN pip install --upgrade pip && \
```

Comment:
Can we use Python 3 here, as well?

Reply:
This does use Python 3. There is no Python 2 in that image.